ci-operator/config/openshift/installer/openshift-installer-release-4.18.yaml

```diff
@@ -374,6 +374,15 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-proxy
   timeout: 6h0m0s
+- always_run: false
```

Comment:
Similar changes need to go into https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-master.yaml.